integration-tests/tests/message_routing.rs

```diff
@@ -140,6 +143,24 @@ fn setup_test_case() -> (Protocol, Chain, Chain, Vec<Worker>) {
             key_pair: test_utils::generate_key(1),
         },
     ];
+    let min_worker_bond = Uint128::new(100);
+    let response = protocol.app.execute_contract(
+        protocol.governance_address.clone(),
+        protocol.service_registry_address.clone(),
+        &service_registry::msg::ExecuteMsg::RegisterService {
+            service_name: protocol.service_name.clone().to_string(),
+            service_contract: Addr::unchecked("nowhere"),
+            min_num_workers: 0,
+            max_num_workers: Some(100),
+            min_worker_bond: min_worker_bond.clone(),
+            bond_denom: AXL_DENOMINATION.into(),
+            unbonding_period_days: 10,
+            description: "Some service".into(),
+        },
+        &[],
+    );
+    assert!(response.is_ok());
```

Comment:
I like the approach that's taken elsewhere to isolate these simple tasks into functions with easy to understand names (see test_utils::register_workers below). Do that for this call too?